### PR TITLE
New structure for the Tax section

### PR DESCRIPTION
**Pull request by jeremyfourna**
## PR Description

Changed the structure of the Tax Management section to show Avalara and Spryker Tax.
A hero picture must be uploaded for Avalara (I added it inside the Avalara folder).

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/pbc/all/tax-management/202212.0/tax-management.md [hunk 1]
```diff
@@ -4,6 +4,43 @@ description: Tax Management PBC lets you define tax rates for different stores a
...
 
-Tax Management lets you manage taxes per store, product, and country.
+Adhere to the respective tax regulations in the countries you sell and effortlessly configure and manage tax rates for products, shipments, and additional services.
 
-The capability consists of a base shop and the marketplace addon. The base shop features are needed for running a regular shop in which your company is the only entity fulfilling orders. To run a marketplace, the features from both the base shop and the marketplace addon are required.
+## Available Tax Providers
+
+| Tax Provider | Build & Maintened by | Available From | Business Model Supported | Suitable When | Can be Extended & Customized |
+| --- | --- | ---| ---| --- | -- |
+| [Avalara](/docs/pbc/all/tax-management/{{page.version}}/avalara/avalara.html) | Spryker | Composer | B2C | You need a robust tax solution for the North American market and the rest of the world. | From the Avalara back-office, you can customize all the tax rules you need for your business. |
```
Comment (andriitserkovnyi) [inline docs/pbc/all/tax-management/202212.0/tax-management.md:13]: Is Avalara back-office an official name?

Comment (andriitserkovnyi) [reply docs/pbc/all/tax-management/202212.0/tax-management.md:13]: Is it maintained by us as well?

File: docs/pbc/all/tax-management/202212.0/avalara/integrate-avalara.md [hunk 2]
```diff
@@ -0,0 +1,756 @@
+---
+title: Integrate Avalara
```
[the remark this answers is not in the document]
Comment (jeremyfourna) [reply docs/pbc/all/tax-management/202212.0/avalara/integrate-avalara.md:2]: Where are the two places you mention?

[the remark this answers is not in the document]
Comment (andriitserkovnyi) [reply docs/pbc/all/tax-management/202212.0/avalara/integrate-avalara.md:2]: `docs/pbc/all/tax-management/202212.0/avalara/integrate-avalara.md`
`docs/pbc/all/tax-management/202212.0/spryker-tax/base-shop/third-party-integrations/integrate-avalara.md`

[the remark this answers is not in the document]
Comment (jeremyfourna) [reply docs/pbc/all/tax-management/202212.0/avalara/integrate-avalara.md:2]: OOOOOh I guess I forgot to remove it from there. let me fix that

**Pull request closed without merging**